inc/cub3d_render.h

```diff
@@ -16,33 +16,188 @@
 # include "data_structure.h"
 # include <stddef.h>
 
+/**
+ * @brief Init the map translation and the backbuffer for rendering
+ * 
+ * @param[in,out] data data structure
+ * @return t_data* data if successful, NULL otherwise
+ */
 t_data	*cub_init(t_data *data);
+/**
+ * @brief Translate the 2d map extracted from input file into a vector of int
+ * 
+ * @param[in,out] data data structure
+ * @return int 1 OK, 0 otherwise
+ */
 int		cub_translate_map(t_data *data);
+/**
+ * @brief Init cub for rendering
+ * 
+ * @param[in,out] data data structure
+ * @return int 0 OK, error otherwise
+ */
 int		cub3d_init_render(t_data *data);
+/**
+ * @brief Main function to render the game.
+ * data->player.fov * 0.5f is used to calculate the plane for the camera
+ *
+ * @param[in,out] data data structure
+ */
 void	cub_render(t_data *data);
-
+/**
+ * @brief Main loop of rendering, updates player position, 
```

Comment:
GitHub won't let me review multiple lines at once. A better fit for this docstring would be "Main loop function. Handles player update logic and rendering"